Makefile

```diff
@@ -14,6 +14,13 @@ BUILDMODE_darwin_arm64=-buildmode=pie
 
 BUILDER_IMAGE=docker.elastic.co/observability-ci/fleet-server-builder:latest
 
+#Benchmark related targets
+BENCH_BASE ?= benchmark-$(COMMIT).out
+BENCH_NEXT ?=
+BENCHMARK_ARGS := -count=8
+BENCHMARK_PACKAGE ?= ./...
+BENCHMARK_FILTER ?= Bench#
```

Comment:
please add a comment to state that the default for this filter is to only run `Benchmark` test cases in test files and exclude any unit tests

Reply:
Thanks for the review @michel-laterman, 
I pushed a change, let me know if you are ok with it.